web/src/components/molecules/Schema/FieldModal/FieldDefaultInputs/Markdown/index.tsx

```diff
@@ -2,14 +2,19 @@ import React from "react";
 
 import Form from "@reearth-cms/components/atoms/Form";
 import MarkdownInput from "@reearth-cms/components/atoms/Markdown";
+import MultiValueField from "@reearth-cms/components/molecules/Common/MultiValueField";
 import { useT } from "@reearth-cms/i18n";
 
-const MarkdownField: React.FC = () => {
+type Props = {
+  multiple?: boolean;
+};
+
+const MarkdownField: React.FC<Props> = ({ multiple }) => {
   const t = useT();
 
   return (
     <Form.Item name="defaultValue" label={t("Set default value")}>
-      <MarkdownInput />
+      {multiple ? <MultiValueField FieldInput={MarkdownInput} /> : <MarkdownInput />}
```

Comment:
Is there a reason you are passing `MarkdownInput` in to MultiValueField? Since MarkdownInput is an atom AND has no props being passed here to it, it doesn't make sense to be passed here. Just adds complexity

Reply:
@KaWaite , `MultiValueField` is an `antd` `form.item` wrapper to convert this atom into accepting multiple values, it can take `Input`, `Markdown input`, or `Textarea` as a prop.
Do you recommend another approach?

Reply:
I'll try a different approach in my next PR, I'll mention you.